docs/content_management/field_types/field_type_reference/field_type_reference.md

```diff
@@ -49,6 +49,8 @@ Custom Field Types have to be programmed in PHP. However, the built-in Field Ty
 | [SesProfiledata](sesprofiledata.md) | Stores address data for a customer. | No | No |
 | [SesSelection](sesselection.md) | Stores a single selection choice based on options from a YAML file. | Yes | Yes |
 | [SpecificationsType](specificationstype.md) | Stores a structured list of specification data for products. | Yes | Yes |
+| [TaxonomyEntry](taxonomyentryfield.md) | Stores one taxonomy entry (tag or product type). | Yes | No |
+| [TaxonomyEntryAssignment](taxonomyentryassignmentfield.md) | Controlls and configures taxonomy. | Yes | No |
```

Comment:
1. It's searchable in both Solr and ES
2. In my opinion, the summary should emphasis on the fact it's used to bridge content with Taxonomy feature, something along the lines of "Makes content taggable by Taxonomy"